.pylintrc

```diff
@@ -19,8 +23,6 @@ disable=
  attribute-defined-outside-init,
  broad-except,
  consider-using-dict-items,
- consider-using-f-string,
```

Comment:
But we still require 2.7 due to CentOS 7, will this not cause issues?

Reply:
@SpyTec It's in the file twice. This is a redundant line, it already appears earlier in the list (line 18):

https://github.com/oamg/convert2rhel/blob/d182c47b5e457c415805568b6b4e881034d6a8d9/.pylintrc#L15-L24

Reply:
Ohh gotcha. Thanks for letting me know!